openlibrary/plugins/upstream/account.py

```diff
@@ -20,6 +20,7 @@
 from openlibrary.i18n import gettext as _
 from openlibrary.core import helpers as h, lending
 from openlibrary.plugins.recaptcha import recaptcha
+from openlibrary.plugins import openlibrary as olib
```

Comment:
Let's not abbreviate unless we really need to, i.e. not here. The few characters saved is not worth the loss of readability as well as codebase searchability.

Reply:
Needs to be changed farther down on the page as well.

Reply:
sounds good, main objective was to prevent a namespace collision (I couldn't import BadRequest directly from openlibrary.plugins.code because of a circular reference, but in this case maybe I should just move the import to that json login function). Good feedback, will implement and patch tomorrow